create-custom-map-style/tileserver/downloadSpritesGlyphs.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Ohhhhhhh. This is awkward, but I don't see a way around it right now. In theory, `tessera` should have glyph/sprite endpoints, but that's never been necessary before. Or, we leave the URLs as-is, but then Maputnik doesn't know how to sign those requests...

Mark this one down as a wart. Depending where in the world they're trying to style, there may be missing glyphs for extended characters.

Reply:
Maputnik will not load a map without a sprite and glyph endpoint. My idea was to use `aws4` and download these as files to a local directory and then serve them using `http-server`... or just use the ones that come with Maputnik